packages/api/src/index.ts

```diff
@@ -0,0 +1,15 @@
+import fastify from "fastify";
+
+const server = fastify();
```

Comment:
why `fastify` and not `express`?
My take would be to use `express` as the code would not be much different, it's more standard AFAIK and we know it better so it could help for the review